Sources/DistributedActors/DistributedActors.docc/ClusterSingleton.md

```diff
@@ -0,0 +1,126 @@
+# ClusterSingleton
+
+Allows for hosting a _single unique instance_ of a distributed actor within the entire distributed actor system, 
+including automatic fail-over when the node hosting the instance becomes down. 
+
+## Overview
+
+A _cluster singleton_ is a conceptual distributed actor that is guaranteed to have at-most one instance within the cluster system among all of its ``Cluster/MemberStatus/up`` members.
+
+> Note: This guarantee does not extend to _down_ members; because a down member is not part of the cluster anymore, and 
```

Comment:
```suggestion
> Note: This guarantee does not extend to _down_ members, because a down member is not part of the cluster anymore, and 
```

Reply:
and...

Reply:
hah finishing it all up now :) thanks for all the fixups, accepting now :)